test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -675,7 +675,7 @@ public void LdaTopicModel()
             var est = data.MakeNewEstimator()
                 .Append(r => (
                     r.label,
-                    topics: r.text.ToBagofWords().ToLdaTopicVector(numTopic: 3, numSummaryTermPerTopic:5, alphaSum: 10, onFit: m => ldaSummary = m.LdaTopicSummary)));
+                    topics: r.text.ToBagofWords().ToLatentDirichletAllocationTopicVector(numberOfTopics: 3, numberOfSummaryTermsPerTopic:5, alphaSum: 10, onFit: m => ldaSummary = m.LdaTopicSummary)));
```

Comment:
>ToLatentDirichletAllocationTopicVector [](start = 50, length = 38)

It's not what I care much about Static pipeline right now, but I would call it `LatentDirichletAllocation` same as we use in Dynamic pipeline.

Reply:
Why not. :)

---
In reply to: [263972927](https://github.com/dotnet/machinelearning/pull/2890#discussion_r263972927) [](ancestors = 263972927)